components/ws-manager-bridge/src/bridge.ts

```diff
@@ -408,7 +408,7 @@ const mapPortVisibility = (visibility: WsManPortVisibility | undefined): PortVis
     }
 };
 
-const mapPortProcotol = (protocol: WsManPortProtocol | undefined): PortProtocol | undefined => {
+const mapPortProtocol = (protocol: WsManPortProtocol | undefined): PortProtocol | undefined => {
```

Comment:
Would you be able to update the signature to remove the undefined? Based on the implementation, it always returns a value